homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -786,7 +787,11 @@ def __init__(self, name, device, entry, unique_id):
             self._device_features = FEATURE_FLAGS_AIRPURIFIER_2S
             self._available_attributes = AVAILABLE_ATTRIBUTES_AIRPURIFIER_2S
             self._speed_list = OPERATION_MODES_AIRPURIFIER_2S
-        elif self._model == MODEL_AIRPURIFIER_3 or self._model == MODEL_AIRPURIFIER_3H:
+        elif (
```

Comment:
```suggestion
        elif self._model in MODELS_PURIFIER_MIOT:
```

Reply:
Thanks syssi, your suggestion looks cleaner. One of my concerns is we are not sure if all of the future miot devices will share the same future set. Maybe we can take actions when that happens...